### DISCUSSION TITLE
Anyone using AudiogridderServer on Parallels?

### DISCUSSION THREAD

**Discussion by steveschow** [apohl79/audiogridder#963] [Help]
I am trying to use Parallels 17.x to host windows10 and audiogridderserver.  On the DAW side I am using MacOS 12.3.1 (Monterey), and LogicPro.

Due to the way that Parallels handles Retina monitors and the Windows resolution settings, which is not entirely clear to me; The GUI from my plugins come through in the AudioGridderServer plugin window super tiny...unreadable really.  Also, the mouse clicks on the ffmpeg DAW side don't seem to line up with the actual GUI...and it becomes unresponsive very easily.  The border around the plugin window on both the windows side and DAW side is much larger then the actual plugin GUI itself.

The main plugin I am trying to use is Razoon Jamstix (there is a free version).  This plugin can have its GUI zoomed.  So I set its zoom to 200%, which makes the GUI big enough to use, but the plugin border is ginormous that way and still the mouse doesn't seem to work from the DAW side...it might be receiving the clicks somehow I'm still not sure, but the screen doesn't update itself anymore. And if another window on the audiogridder server side goes in front of the plugin window, then DAW side will see that other junk, like a browser window or whatever.

Well just wondering if anyone has gotten audiogridder working from a VM such as parallels and if so...how did you set it up to work correctly.  This is a decent potential way to host Win64 plugins on a Mac, without a seperate PC needed, presuming we can get it to work properly end to end.

**Comment by MushPsykotik** [comment]
Hi,

I dont have answer because i dont use a Mac. But if your goal is to run AG server for windows on a mac, it run like a charm trough Wine. If on the same machine that your DAW, you'll probably found the server at _localhost:55056_ address. I guess using a virtual desktop could be useful.

**Reply by jahudka** [reply]
Hi @MushPsykotik (or anyone!), can you please offer any tips or resources on how to get AG server for Win running on a Mac? I've tried both the stable and the beta and I can get neither to start - they both fail with the same error, something like this:

```
Unhandled page fault on read access to 0000000000000020 at address 00000001409700BE (thread 0140)
```

I've tried reinstalling Wine from Homebrew as per WineHQ instructions, but no dice. I'm on Wine 7.0 and MacOS Big Sur. Any ideas?

**Reply by jahudka** [reply]
Almost a year + a major wine release + a major macOS release later and the result is still the same :'( anyone have any ideas for running AudioGridder server for Windows under wine on macOS?

**Comment by steveschow** [comment]
I have used Wine a lot in the past.  Good to know audiogridder runs on wine, but If I was going to run Wine I probably would use WineASIO instead of audiogridder.  In any case I'm on MacOS 12.3 and it does not support 32bit.  I have tried using Wine with 64bit stuff in the past with very limited success.  I will stick with Parallels full VM for now.  Thanks.

**Comment by apohl79** [comment]
I'm using parallels during development. The scaling of plugins is not consistent on windows unfortunately. Some plugins work with higher scaling factors, some just get very tiny. The best is usually to set windows scaling to 100% (parallels sets it to 120% on my machine).

You can also zoom the AG plugin UI, but i guess thats all you can do.

**Reply by steveschow** [reply]
I guess this is really more a problem with plugins that must not be hiDPI capable.  Maybe.  Anyway, with Parallels, I can use the Retina mode, which is about a zillion times nicer on my Mac Retina display for generally using windows under Parallels.  But then in windows everything needs to be scaled up otherwise it will be tiny unless its a HiDPI aware software.  Certain apps that aren't smart enough about windows scaling will simply look too tiny to be usable.  Parallels has a scaled mode, which I tried initially and will try some more, but it got out of wack in some way in audiogridder, the GUI coming across to the Mac doesn't line up, so its not responsive to mouse clicks, and so forth.  I will try it again...even if it makes the overall windows experience scaled and grainy and much worse then Retina mode...if it causes Rayzoon Jamstix to work correctly, that's all I really care about.
